style.css

```diff
@@ -705,6 +726,174 @@ ul {
     justify-content: center;
     align-items: center;
   }
+
+  /* code for pop window */
+  .overlay {
+    z-index: 43;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 16px;
+    gap: 24px;
```

Comment:
* Kindly adjust your popup styling to suits the figma design below are the highlighted section that needs adjustment 
## Figma design 
![image](https://github.com/Shahid-Bagwan/Portfolio/assets/52497246/1d16ced2-686e-4aeb-a9bc-0b51a84bcc53)
![image](https://github.com/Shahid-Bagwan/Portfolio/assets/52497246/f363a434-e486-453f-8838-cbe8340ed8e1)
## Yours 
![image](https://github.com/Shahid-Bagwan/Portfolio/assets/52497246/3cf3808a-dd83-4434-aeee-948aa2fc1b7a)
![image](https://github.com/Shahid-Bagwan/Portfolio/assets/52497246/9b5c3843-36b9-4737-bfea-c771a4ce0557)